src/client/components/pages/Courses/CoursesPage.tsx

```diff
@@ -408,19 +411,30 @@ const CoursesPage: FunctionComponent = (): ReactElement => {
                   onChange={toggleColumn}
                 />
               </ViewModal>
-              <Button
-                variant={VARIANT.INFO}
-                forwardRef={customizeViewButtonRef}
-                onClick={() => {
-                  setViewModalVisible(true);
-                }}
-              >
-                <FontAwesomeIcon
-                  icon={faWrench}
+              <MenuFlex>
+                <Button
+                  variant={VARIANT.INFO}
+                  forwardRef={customizeViewButtonRef}
+                  onClick={() => {
+                    setViewModalVisible(true);
+                  }}
+                >
+                  <FontAwesomeIcon
+                    icon={faWrench}
+                  />
+                  {' '}
+                  Customize View
+                </Button>
+                <Checkbox
+                  id="showRetiredCheckbox"
+                  name="showRetiredCheckbox"
+                  label="Show Retired"
+                  checked={showRetired}
+                  onChange={() => setShowRetired(!showRetired)}
+                  labelPosition={POSITION.LEFT}
```

Comment:
Minor point, but in other places where we've used checkboxes in the app we've used `POSITION.RIGHT`. Can we change that here so we're consistent?